### DISCUSSION TITLE
Github pages site never finishes building

### DISCUSSION THREAD

**Discussion by not-adri** [community/community#81764] [Other Feature Feedback, Questions, & Ideas]
"Your GitHub Pages site is currently being built from the main branch" Has been going on for over 30 minutes. Any ideas what what could be wrong 

url: https://github.com/not-adri/yesorno

**Comment by yoannchaudet** [comment]
I don't think we automatically refresh the banner.

But since the builds happen in Actions, you can lookup the details of what's going on by heading over to: https://github.com/not-adri/yesorno/actions.
